src/main/scala/dpla/ingestion3/mappers/providers/OhioMapping.scala

```diff
@@ -129,20 +133,35 @@ class OhioMapping extends Mapping[NodeSeq] with XmlExtractor with IdMinter[NodeS
     }).headOption
   }
 
-  override def isShownAt(data: Document[NodeSeq]): EdmWebResource =
-    uriOnlyWebResource(
-      Utils.createUri(extractStrings(data \ "metadata" \\ "isShownAt")
-        .headOption
-        .getOrElse(
-          throw new RuntimeException(s"No isShownAt property in record ${getProviderId(data)}")
-        )))
+  override def isShownAt(data: Document[NodeSeq])
+                        (implicit msgCollector: MessageCollector[IngestMessage]): EdmWebResource =
+    extractStrings(data \ "metadata" \\ "isShownAt").flatMap(uriStr => {
+      Try { new URI(uriStr)} match {
+        case Success(uri) => Option(uriOnlyWebResource(uri))
+        case Failure(f) =>
+          msgCollector.add(
+            mintUriError(id = getProviderId(data), field = "isShownAt", value = uriStr))
+          None
+      }
+    }).headOption match {
+      case None => msgCollector.add(missingRequiredError(getProviderId(data),"isShownAt")) // record error message
+        uriOnlyWebResource(new URI("")) // FIXME model should be changed to not compel empty EdmWebResource
```

Comment:
I think this would be fatal too.

Reply:
Yeah I need to change the level of these errors and bump it to fatal